packages/vanilla/src/sass/components/tabs/_index.scss

```diff
@@ -0,0 +1,58 @@
+.cbp-tabset {
+  display: flex;
+  align-items: flex-end;
+  //gap: var(--cbp-space-2x);
+  border-bottom: var(--cbp-border-size-md) solid var(--cbp-color-gray-cool-30);
+  margin-bottom: var(--cbp-space-4x);
```

Comment:
I don't think we need this, we can let the content of whatever it's going to have pad/margin itself.

Reply:
Almost nothing has a default top margin/padding, so it won't - the content will butt right up to the bottom of the tabs, which looks bad. I find this preferable to putting the styling on the tab panels. There were no specs on this, so I'd rather revisit this when the specs get updated. This wouldn't apply for vertical tabs either, so we may need to tweak things when we allow for both as well. But that change would affect the tabset, not the panels, which goes with my reasoning for putting it there.